include/engine/api/base_api.hpp

```diff
@@ -56,8 +56,8 @@ class BaseAPI
             // TODO: check forward/reverse
             return json::makeWaypoint(
                 phantom.location,
-                util::coordinate_calculation::fccApproximateDistance(phantom.location,
-                                                                     phantom.input_location),
+                util::coordinate_calculation::greatCircleDistance(phantom.location,
```

Comment:
Renamed it to more generic `greatCircleDistance`, but have no strong opinion here - we can leave old naming(or introduce new one, may be just `distance`?).

Reply:
And btw it cheap-ruler is no longer based on FCC formula https://twitter.com/mourner/status/1560003819211309056?s=20&t=6zvluLg1gn3HYB0zM1znOw